packages/api/src/OsdkObjectFrom.ts

```diff
@@ -158,14 +158,6 @@ export type GetPropsKeys<
   : IsAny<P> extends true ? PropertyKeys<Q>
   : P;
 
-export type GetProps<
```

Comment:
I deleted because we didn't need it right now, but I can leave it in case we want it later?

Reply:
Yeah thats fine